src/crimson/osd/ops_executer.cc

```diff
@@ -489,6 +489,11 @@ OpsExecuter::list_snaps_iertr::future<> OpsExecuter::do_list_snaps(
   const ObjectState& os,
   const SnapSet& ss)
 {
+  if (msg->get_snapid() != CEPH_SNAPDIR) {
+    logger().error("LIST_SNAPS with incorrect context");
```

Comment:
No need to log at error here -- it's not actually a problem for the OSD.

Reply:
yes, should be debug level similar to classical osd